ulid.gemspec

```diff
@@ -17,5 +17,5 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'sysrandom', '>= 1.0.0', '< 2.0'
+  spec.add_dependency 'sysrandom', '>= 1.0.0', '< 2.0' unless RUBY_VERSION >= '2.5'
```

Comment:
what about omit the negation ?
`spec.add_dependency 'sysrandom', '>= 1.0.0', '< 2.0' if RUBY_VERSION < '2.5'`

Reply:
Right. It is much straightforward, i'll fix it.

Reply:
done in 9162f3a